frontend/test/__support__/ui.tsx

```diff
@@ -13,28 +13,22 @@ import { state as sampleDatabaseReduxState } from "__support__/sample_database_f
 import type { User } from "metabase-types/api";
 import type { State } from "metabase-types/store";
 
-import { createMockUser } from "metabase-types/api/mocks";
 import { createMockState } from "metabase-types/store/mocks";
 
+import mainReducers from "metabase/reducers-main";
+import publicReducers from "metabase/reducers-public";
+
 import { getStore } from "./entities-store";
 
 export interface RenderWithProvidersOptions {
   currentUser?: User;
```

Comment:
@kulyk should it be removed too? e.g. use `storeInitialState` directly

Reply:
Oops, looks like I forgot to update the type. Thanks!